build/application/src/main/resources/application.properties

```diff
@@ -32,4 +37,17 @@ management.metrics.mongo.command.enabled=false
 management.metrics.mongo.connectionpool.enabled=false
 
 spring.devtools.restart.poll-interval=6s
-spring.devtools.restart.quiet-period=5s
\ No newline at end of file
+spring.devtools.restart.quiet-period=5s
+
+spring.boot.admin.context-path=spring-admin
+spring.boot.admin.ui.title=${spring.application.name} Admin
+spring.boot.admin.ui.brand=<img src="/services/web/resources/images/dirigible.svg"><span>Eclipse Dirigible Admin</span>
+
+spring.boot.admin.client.url=http://localhost:${server.port}/spring-admin
```

Comment:
Will this work, if the instance is deployed somewhere else, instead of `localhost`?

Reply:
I guess so, let's try it tomorrow on k8s

Reply:
verified on k8s